test/byContract/IexecEscrow/IexecEscrowToken.test.ts

```diff
@@ -0,0 +1,473 @@
+            const withdrawToArgs = Object.values(withdrawToParams) as [BigNumber, string];
+
+            expect(await iexecPocoAsAccountA.callStatic.withdrawTo(...withdrawToArgs)).to.be.true;
+            await expect(iexecPocoAsAccountA.withdrawTo(...withdrawToArgs))
+                .to.changeTokenBalances(
+                    rlcInstance,
+                    [iexecPoco, withdrawToParams.target],
+                    [-withdrawToParams.amount, withdrawToParams.amount],
+                )
+                .to.emit(rlcInstance, 'Transfer')
+                .withArgs(iexecPoco, withdrawToParams.target, withdrawToParams.amount)
+                .to.changeTokenBalances(iexecPoco, [accountA], [-standardAmount])
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(accountA.address, AddressZero, withdrawToParams.amount);
+        });
+        it('Should withdraw To with zero token', async () => {
+            const withdrawToParams = {
+                amount: 0,
+                target: accountB.address,
+            };
+            const withdrawToArgs = Object.values(withdrawToParams) as [BigNumber, string];
+
+            expect(await iexecPocoAsAccountA.callStatic.withdrawTo(...withdrawToArgs)).to.be.true;
+            await expect(iexecPocoAsAccountA.withdrawTo(...withdrawToArgs))
+                .to.changeTokenBalances(rlcInstance, [iexecPoco, accountB], [-0, 0])
+                .to.emit(rlcInstance, 'Transfer')
+                .withArgs(iexecPoco, withdrawToParams.target, 0)
+                .to.changeTokenBalances(iexecPoco, [accountA], [-0])
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(accountA.address, AddressZero, 0);
+        });
+        it('Should not withdraw To tokens with empty balance', async () => {
+            await expect(
+                iexecPocoAsAccountA.withdrawTo(standardAmount, accountB.address),
+            ).to.be.revertedWithoutReason();
+        });
+        it('Should not withdraw To tokens with insufficient balance', async () => {
+            await rlcInstanceAsAccountA
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+            await iexecPocoAsAccountA.deposit(standardAmount).then((tx) => tx.wait());
+
+            await expect(
+                iexecPocoAsAccountA.withdrawTo(standardAmount.mul(2), accountB.address),
+            ).to.be.revertedWithoutReason();
+        });
+    });
+
+    describe('Recover', () => {
+        it('Should recover from balance deviation', async () => {
+            await rlcInstance.connect(iexecAdmin).transfer(proxyAddress, standardAmount); // Simulate deviation
+
+            const initTotalSupply = await iexecPoco.totalSupply();
+            const expectedDelta = standardAmount;
+
+            await expect(iexecPocoAsAdmin.recover())
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(AddressZero, iexecAdmin.address, expectedDelta);
```

Comment:
Add `changeTokenBalances` in these methods

Reply:
[add changeTokenBalances checks on recover test](https://github.com/iExecBlockchainComputing/PoCo/pull/141/commits/ad319605cc7fb51e80ea94ae714e23e8ec7b64d5)